src/msw/renderer.cpp

```diff
@@ -604,6 +604,11 @@ int wxRendererMSW::GetHeaderButtonHeight(wxWindow * WXUNUSED(win))
 
     wxON_BLOCK_EXIT1( ::DestroyWindow, hwndHeader );
 
+    // Must ensure the proper font is set or the wrong value will be returned
+    // At 200% scaling it was returning a height of 28 when it should have been 40
+    WXHANDLE hFont = (win && win->GetFont().IsOk() ? win->GetFont() : wxSystemSettings::GetFont(wxSYS_DEFAULT_GUI_FONT)).GetResourceHandle();
+    ::SendMessage(hwndHeader, WM_SETFONT, (WPARAM)hFont, MAKELPARAM(TRUE, 0));
```

Comment:
Do we really need to pass `TRUE` (forcing the control redraw) here? Looks like it shouldn't be needed for this (invisible) window.